package.json

```diff
@@ -5,10 +5,47 @@
   "author": "Valera <Koniushok@users.noreply.github.com>",
   "private": true,
   "dependencies": {
-    "webpack": "^4.28.3",
-    "webpack-cli": "^3.2.1"
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "react-hot-loader": "^4.6.3",
+    "styled-components": "^4.1.3",
+    "webpack": "^4.28.3"
```

Comment:
Are you sure you need webpack in dependencies instead devDependencies?

Reply:
eslint throw error when webpack in devDependencies. But I can ignore it .
"import/no-extraneous-dependencies": ["error", {"devDependencies": true}]

Reply:
thanks I got it

Reply:
I didn't use validate package.json.